saslauthd/lak.c

```diff
@@ -1716,6 +1716,15 @@ static int lak_check_password(
 #define EVP_MD_CTX_new()      EVP_MD_CTX_create()
 #define EVP_MD_CTX_free(ctx)  EVP_MD_CTX_destroy((ctx))
 
+static void *OPENSSL_zalloc(size_t num)
```

Comment:
do we need to ifdef this? I guess this would be redefined with new versions of openssl?